nautobot/core/testing/forms.py

```diff
@@ -0,0 +1,28 @@
+from django.test import tag, TestCase
+
+from nautobot.core.forms.fields import DynamicModelChoiceField
+from nautobot.core.utils.lookup import get_filterset_for_model
+
+
+# TODO(timizuo): All Form Test cases should inherit from FormTestCases
+@tag("unit")
+class FormTestCases:
+    class BaseFormTestCase(TestCase):
+        """Base class for generic form tests."""
+
+        form_class = None
+
+        def test_form_dynamic_model_choice_fields_query_params(self):
+            for field_name, fields_class in self.form_class.declared_fields.items():
+                if not isinstance(fields_class, DynamicModelChoiceField):
```

Comment:
Note that `DynamicModelMultipleChoiceField` is not actually a subclass of `DynamicModelChoiceField`. Should we test for both, perhaps by `isinstance(fields_class, DynamicModelChoiceMixin)` instead?